content/posts/2022-08-17-ocp-on-m1.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Running ocp on M1 machines"
+date: 2022-08-17T08:58:10+05:30
+---
+
+From the 2.4.1 release, CRC has been running on macOS aarch64 M1 CPUs.
+However, the support was limited to podman preset on M1. We've been asked numerous times when CRC openshift
+preset would get M1 support. This has changed with the recently released [crc 2.7.1](https://github.com/code-ready/crc/releases/tag/v2.7.1).
```

Comment:
Numerous times we received requests if it would be possible to instead offer an OpenShift that would run on these machines.